### DISCUSSION TITLE
Is there a size limit in solving the robust problem?

### DISCUSSION THREAD

**Discussion by NewWalker2020** [yalmip/YALMIP#1268] [General discussion]
When I try to solve a MIP robust problem setting uncertain variable with the "uncertain" command in yalmip, it seems having some trouble in model transformation. I guess that yalmip may have a size limit in solving the robust problem diretly. If so, can u tell me the maximum scale of the robust problem that yalmip supports?  or how can I avoid this problem?
It has been stuck in this step for a day! I am a greenhand in yalmip and don't konw what to do next >.<
![robust problem](https://user-images.githubusercontent.com/64523019/182383417-b3f73cc9-3d48-4e2c-a9aa-9d149f469b92.png)
![robust](https://user-images.githubusercontent.com/64523019/182384112-62fd7cd7-75b2-4fe0-8756-ad3c06f48186.png)

**Comment by NewWalker2020** [comment]
Can I ask there? or in discussions? Sorry, it's my first time to ask.

**Reply by johanlofberg** [reply]
Completely missed your question. I guess it has been resolved now, but the answer is that YALMIP as such does not have any limitations, but robust optimization can lead to exponentially sized problems so even a small model can cause the problem to be intractable in practice.
